docs/api/renderers/WebGLRenderer.html

```diff
@@ -225,6 +225,14 @@
 		<h3>[property:Boolean shadowMap.enabled]</h3>
 		<div>If set, use shadow maps in the scene. Default is *false*.</div>
 
+		<h3>[property:Boolean shadowMap.autoUpdate]</h3>
+		<div>Enables automatic updates to the shadows in the scene. Default is *true*.</div>
+		<div>If you do not require dynamic lighting / shadows, you may set this to false after the first *render* call.</div>
```

Comment:
I don't think you need to set it to false after the first render call. You can set it to false when the renderer is instantiated.
```javascript
renderer.shadowMap.autoUpdate = false;
renderer.shadowMap.needsUpdate = true;
```

Reply:
Done